esterni/AnalisiDeiRequisiti/res/sections/requisiti.tex

```diff
@@ -26,139 +26,217 @@ \subsection{Requisiti funzionali}
-			R1F6.2&L'amministratore deve poter visualizzare in forma tabellare le occupazioni di una postazione specifica, con date e orari di inizio e fine delle occupazioni e nomi e cognomi dei dipendenti.	&Capitolato - UC6.2 - Verbale esterno 2021-01-04	\\
-			\hline
-			R1F6.2.1&L'amministratore deve poter effettuare una ricerca tra le occupazioni di una specifica postazione indicando il periodo all'interno del quale l'utente occupava la postazione	& 	Capitolato - UC6.2.1 - Verbale esterno 2021-01-04\\
-			\hline
-			R1F6.3 & L'amministratore deve poter scaricare il report delle occupazioni che sta visualizzando in un formato leggibile & Capitolato - UC6.3 - Verbale esterno 2021-01-04\\
-			\hline
-			R2F6.3.1 & L'amministratore deve poter scaricare il report delle occupazioni che sta visualizzando nel formato PDF & Capitolato - UC6.3.1 - Verbale esterno 2021-01-04\\
-			\hline
-R2F7&L'amministratore deve poter ottenere una tabella delle sanificazioni di tutte le postazioni, con data e ora in cui sono avvenute, nome e cognome di chi le ha eseguite e il suo ruolo.	&Capitolato - UC7 - Verbale esterno 2021-01-04	\\
-					
-					\hline
-R2F7.1&L'amministratore deve poter scaricare il report delle sanificazioni che sta visualizzando in un formato leggibile.	&Capitolato - UC7.1 Verbale esterno 2021-01-04	\\
-
-\hline
-R2F7.1.1&L'amministratore deve poter scaricare il report delle sanificazioni che sta visualizzando nel formato PDF.	&Capitolato - UC7.1.1 - Verbale esterno 2021-01-04	\\
-
-\hline
-					R1F8&Il dipendente deve poter visualizzare lo stato di una postazione, tramite tag RFID.	&Capitolato - UC8 	\\
-					\hline
-			R1F9&Il dipendente deve poter segnalare la propria presenza su una postazione in tempo reale, tramite tag RFID.	&Capitolato - UC9 	\\
-		\hline
-		R1F9.1&Il dipendente deve poter registrare i momenti di inizio e di fine della sua occupazione di una postazione, tramite Ethereum.	&Interna - UC9.3 	\\
+		R1F2	&	L'utente non autenticato deve potersi autenticare nel webserver inserendo come credenziali: nome utente e password & Capitolato - UC2	\\
+		\hline
+		R1F2.1	&	L'utente non autenticato deve ricevere un messaggio di errore nel caso abbia sbagliato a inserire le credenziali nel web server& Interna - UC2.1	\\
+		\hline
+		R1F2.2	&	L'utente non autenticato che stia provando ad accedere al web server deve poter ricevere un messaggio di errore nel caso il suo account sia disabilitato& Interna - UC2.2	\\
+		\hline
+		R1F3	&	L'utente non autenticato deve potersi autenticare nell'applicazione mobile inserendo come credenziali nome utente e password & Capitolato - UC3	\\
+		\hline
+		R1F3.1	&	L'utente non autenticato deve ricevere un messaggio di errore nel caso abbia sbagliato a inserire le credenziali nell'applicazione mobile& Interna - UC3.1	\\
+		\hline
+		R1F3.2	&	L'utente non autenticato che stia provando ad accedere al web server deve poter ricevere un messaggio di errore nel caso il suo account sia disabilitato& Interna - UC3.2	\\
```

Comment:
modificare `L'utente non autenticato che stia provando ad accedere al web server` in `L'utente non autenticato che stia provando ad accedere all'applicazione mobile`